### PR TITLE
Fix/large fw nrf54l15

### PR DESCRIPTION
**Pull request by kylebonnici**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 4.4.3 - UNRELESED
+
+### Fixed
+
+-   Issue with flashing firmware on nRF54L15 larger than 1 MB firmware.
+-   Disable unsupported firmware read for nRF54L Family.
```
Comment (greg-fer) [inline Changelog.md:6]:
```suggestion
### Changed

-   Disabled unsupported firmware read for nRF54L Series.

### Fixed

-   Issue with programming firmware on nRF54L15 when it is larger than 1 MB.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]

**Comment by greg-fer** [comment]
What should we do about the table in https://docs.nordicsemi.com/bundle/nrf-connect-programmer/page/index.html ? Slava said there are issues with programming nRF54L in another task, and now we're adding entries for nRF54L as if it is supported all right.
We need to update that table.

File: Changelog.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+## 4.4.3 - UNRELESED
```
Comment (KievDevel) [inline Changelog.md:1]:
```suggestion
## 4.5.0 - UNRELESED
```

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "pc-nrfconnect-programmer",
-    "version": "4.4.2",
+    "version": "4.4.3",
```
Comment (KievDevel) [inline package.json:3]:
```suggestion
    "version": "4.5.0",
```

File: Changelog.md [hunk 4]
```diff
@@ -1,3 +1,13 @@
+## 4.5.0 - UNRELEASED
```
Comment (KievDevel) [inline Changelog.md:1]:
```suggestion
## 4.5.0 - 2024-12-17
```

**Pull request merged**